modules/update-using-custom-machine-config-pools-about.adoc

```diff
@@ -23,13 +23,6 @@ Do not remove the default worker label from the nodes. The nodes *must* have a r
 ====
 
 . Pause the MCPs you do not want to update as part of the update process.
-+
-[NOTE]
-====
-Pausing the MCP also pauses the `kube-apiserver-to-kubelet-signer` automatic CA certificates rotation. New CA certificates are generated at 292 days from the installation date and old certificates are removed 365 days from the installation date. See the link:https://access.redhat.com/articles/5651701[Understand CA cert auto renewal in Red Hat OpenShift 4] to find out how much time you have before the next automatic CA certificate rotation. 
```

Comment:
The linked article https://access.redhat.com/articles/5651701 is also incorrect as of the latest version, we should also update that although that is out of the scope of this PR